<ui version="4.0" >
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>218</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>418</width>
    <height>218</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>418</width>
    <height>218</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Export</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="pmchart.qrc" >:/images/document-export.png</iconset>
  </property>
  <property name="toolTip" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="formatTextLabel" >
           <property name="text" >
            <string>Image Format</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="formatComboBox" >
           <property name="maxCount" >
            <number>32</number>
           </property>
           <property name="duplicatesEnabled" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="transparentCheckBox" >
         <property name="text" >
          <string>Transparent Background</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QRadioButton" name="selectedRadioButton" >
           <property name="text" >
            <string>Selected Chart Only</string>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="allChartsRadioButton" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>1</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>All Charts in Tab</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>0</number>
       </property>
       <item row="1" column="1" >
        <widget class="QSpinBox" name="heightSpinBox" >
         <property name="suffix" >
          <string> px</string>
         </property>
         <property name="maximum" >
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="qualityLabel" >
         <property name="text" >
          <string>Quality</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QSpinBox" name="qualitySpinBox" >
         <property name="suffix" >
          <string>%</string>
         </property>
         <property name="maximum" >
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="QSlider" name="qualitySlider" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="widthLabel" >
         <property name="text" >
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="heightLabel" >
         <property name="text" >
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QSpinBox" name="widthSpinBox" >
         <property name="suffix" >
          <string> px</string>
         </property>
         <property name="maximum" >
          <number>999999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="fileLabel" >
       <property name="text" >
        <string>File</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileLineEdit" >
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filePushButton" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="pmchart.qrc" >:/images/document-export.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>214</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string/>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pmchart.qrc" />
 </resources>
</ui>
